checkov/terraform/parser.py

```diff
@@ -425,18 +427,21 @@ def _load_modules(self, root_dir: str, module_loader_registry: ModuleLoaderRegis
                             logging.info(f'Got no content for {source}:{version}')
                             continue
 
-                        self._internal_dir_load(directory=content.path(),
-                                                module_loader_registry=module_loader_registry,
-                                                dir_filter=dir_filter, specified_vars=specified_vars,
-                                                module_load_context=module_load_context,
-                                                keys_referenced_as_modules=keys_referenced_as_modules)
+                        if not self._module_definitions_cache.get(content.path()):
```

Comment:
```suggestion
                        if content.path() not in self._module_definitions_cache:
```
this is a bit faster